openstacksdk_plugin/tests/test_network.py

```diff
@@ -36,8 +39,14 @@ def setUp(self):
         self._ctx = self.get_mock_ctx('TestNetwork')
         current_ctx.set(self._ctx)
 
-    def test_create(self, _cx):
+    @mock.patch(
+        '{0}.create_network'.format(MODULE_PATH),
+        return_value=mock.MagicMock())
+    def test_create(self, _cx, _fn):
```

Comment:
@EarthmanT What do you mean by `_cx` here is it the mocked patched for connect object for openstack ? and the `_fn` is the mocked patched for underlying function for resource ?

Reply:
Yes. _cx = `openstack.connect` _fn = patched API call.